packages/pwa-kit-react-sdk/CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
+## v4.0.0-extensibility-preview.0 (Nov 28, 2024)
```

Comment:
We probably don't need the dev 4.0.0 version that is below this.

Reply:
Done